### PR TITLE
feat: digest auth

### LINKED ISSUE #1094: Try It Auth: Digest-lite
As a user of an API that is protected with HTTP Digest, I need to be able to make requests in Try It even if we don't have a nice UI set up yet.

Similar to when we made OAuth2-lite, can we make sure APIs using HTTP Digest are not blocked through lack of proper support coming in #692. This can just be an Authentication header with the default value of "Digest " and maybe a textarea not an input.

### PR DESCRIPTION
**Pull request by mpodlasin**
Closes #1094

### CODE REVIEW

File: packages/elements-core/src/__fixtures__/operations/put-todos.ts [hunk 1]
```diff
@@ -459,6 +459,14 @@ export const httpOperation: IHttpOperation = {
         bearerFormat: 'Authorization',
       },
     ],
+    [
+      {
+        key: 'digest',
+        type: 'http',
+        scheme: 'digest',
+        description: 'Digest is cool. Use digest. No questions asked money back guarantee.',
```
Comment (marcelltoth) [inline packages/elements-core/src/__fixtures__/operations/put-todos.ts:467]: ❤️

**Review by marcelltoth — COMMENTED** [review]
It's missing a file:)

**Comment by mpodlasin** [comment]
> It's missing a file:)

Huh? What file? 🤔

**Comment by mallachari** [comment]
Mufasa always dies :(

File: packages/elements-core/src/components/TryIt/DigestAuth.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            response="6629fae49393a05397450978507c4ef1",
+            opaque="5ccc069c403ebaf9f0171e9517f40e41"
+`;
+
+export const DigestAuth: React.FC<BasicAuthProps> = ({ onChange, value }) => {
+  return (
+    <Panel.Content className="ParameterGrid">
+      <div>Authorization</div>
+      <Text mx={3}>:</Text>
+      <Flex flex={1}>
```
Comment (marcelltoth) [inline packages/elements-core/src/components/TryIt/DigestAuth.tsx:25]: `Flex` feels like an overkill here, it has a single child.

File: packages/elements-core/src/components/TryIt/DigestAuth.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import { Flex, Panel, Text } from '@stoplight/mosaic';
+import * as React from 'react';
+
+interface BasicAuthProps {
```
Comment (marcelltoth) [inline packages/elements-core/src/components/TryIt/DigestAuth.tsx:4]:
```suggestion
interface DigestAuthProps {
```

**Review by marcelltoth — APPROVED** [review]
Tentative approval, looks good!:)

**Pull request merged**